inst/rmarkdown/templates/ieee/skeleton/skeleton.Rmd

```diff
@@ -75,15 +100,29 @@ with_amsmath: true
 # with_array: true
 # with_dblfloatfix: true
 
+
+## If you're using latex style \cite commands, choose the backend you want to use.
+# natbib: true
+# biblatex: true
+# citation_sorting: none   ## used as sorting option of the biblatex package (if selected)
+
+## If you're using pandoc style @cite commands, these options set the correct
+## citation style, and add links to citations. 
+## Note that this bypasses latex packages that alter citations, e.g. hyperref.
+citation-style: ieee
```

Comment:
It is more common to use the short version `csl` - I prefer we use that

https://bookdown.org/yihui/rmarkdown-cookbook/bibliography.html

```suggestion
csl: http://www.zotero.org/styles/ieee # download the file if you prefer include it locally
```